libecl/src/ecl_kw.c

```diff
@@ -2518,36 +2518,39 @@ bool ecl_kw_is_kw_file(fortio_type * fortio) {
 }
 
 
-
-
+static bool ecl_kw_is_grdecl_file_rewind(FILE * stream, const long int pos, bool ret_val) {
```

Comment:
Use `offset_type` for file offsets instead of naked `long int`. See discussion [here](https://github.com/Ensembles/ert/blob/master/libert_util/src/util_lfs.c)